erpnext/manufacturing/doctype/production_planning_tool/production_planning_tool.py

```diff
@@ -394,7 +394,12 @@ def get_requested_items(self):
 			total_qty = sum([flt(d[0]) for d in so_item_qty])
 			if total_qty > item_projected_qty.get(item, 0):
 				# shortage
-				requested_qty = total_qty - flt(item_projected_qty.get(item))
+				if self.create_material_requests_for_all_required_qty:
```

Comment:
Should this depend on the condition in line 395: `if total_qty > item_projected_qty.get(item, 0):`?